gbpservice/neutron/tests/unit/nfp/configurator/drivers/firewall/test_vyos_fw_driver.py

```diff
@@ -96,9 +96,10 @@ def test_configure_source_routes(self):
             self.driver.configure_routes(
                 self.fo.context, self.kwargs)
 
+            routes_timeout = 60
             mock_post.assert_called_with(self.fo.url_for_add_src_route,
                                          data=self.fo.data_for_add_src_route,
-                                         timeout=self.fo.timeout)
+                                         timeout=routes_timeout)
```

Comment:
I think, Ashutosh added this for having a less timeout. As part of review comments, he is going to remove this in new PR. So, revert this.